### PR TITLE
Add k8s 1.25.0 binaries to VHD.

### PR DESCRIPTION
**Pull request by yizhang4321**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -495,6 +495,7 @@ MULTI_ARCH_KUBE_BINARY_VERSIONS="
 1.23.8-hotfix.20220620
 1.24.0-hotfix.20220615
 1.24.3
+1.25.0
 "
```
Comment (ganeshkumar5699) [inline vhdbuilder/packer/install-dependencies.sh:500]: `crictl` binary for k8s 1.25.0 needs to be cached in the VHD as well. 
The `1.25.0` version needs to be added in line 150 of `install-dependencies.sh` and it needs to be added to crictl version in components.json as well (line 469).

Comment (ganeshkumar5699) [reply vhdbuilder/packer/install-dependencies.sh:500]: Other parts of the PR look good

Comment (ganeshkumar5699) [reply vhdbuilder/packer/install-dependencies.sh:500]: And we also want to confirm that the VHD builds succeed (might need a retry here).

Comment (yizhang4321) [reply vhdbuilder/packer/install-dependencies.sh:501]: crictl 1.25 added

**Comment by ganeshkumar5699** [comment]
The VHD build tests fail during the download step of the "Download Asc baseline". Creating the PR from a branch in Azure/Agentbaker directly, instead of a fork, should make the tests run.

**Comment by alexeldeib** [comment]
@ganeshkumar5699 assuming we can close since we shipped 1.25.2?

**Pull request closed without merging**